packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -2,52 +2,98 @@
-  * `state` &mdash; value to be returned in the token. The client application can use it to remember the state of its interaction with the end user at the time of the authentication call
-* Your application sends a redirect response to the browser containing the composed authentication request. The browser interprets the redirect, and sends the authentication request to the `/authorize` endpoint on the Okta authorization server over secure HTTPS
-* Okta then either sends the browser to an Okta sign-in page which asks the end user to provide authentication and consent to the requested scopes, or confirms that the user is already authenticated
-* The end user provides proof of identity (using any of the supported authentication methods) and gives consent for any requested claims defined in your `scopes` parameter. The claims are specific pieces of information to be included in the token provided from Okta
-* The browser receives an authorization code response from the Okta authorization server. If there is an error or the authentication fails for any reason, the browser returns the error to the end user. If the authentication is successful, the Okta authorization server issues an authorization code value and includes it in a redirect that sends the browser over to the Client at the Login Redirect URI callback location
+  * `scope` &mdash; must contain `openid` to indicate that this is an OIDC request and you want to get back an ID token. You can add optional scopes to your request to limit the user information returned
+  * `state` &mdash; a value returned in the token. The client application can use it to remember the state of its interaction with the end user at the time of the authentication call
+* Your application sends a redirect response to the browser containing the composed authentication request. The browser interprets the redirect and sends the authentication request to the `/authorize` endpoint on the Okta authorization server over secure HTTPS.
+* Okta then sends the browser to an Okta sign-in page which asks the end user to provide authentication and consent to the requested scopes, or confirms that the user is already authenticated.
+* The end user provides proof of identity (using any of the supported authentication methods) and gives consent for any requested claims defined in your `scopes` parameter. These claims are specific pieces of information to be included in the token provided by Okta.
+* The browser receives an authorization code response from the Okta authorization server. If there is an error or the authentication fails for any reason, the browser returns the error to the end user. If the authentication is successful, the Okta authorization server issues an authorization code value and includes it in a redirect that sends the browser over to the Client at the Login Redirect URI callback location.
```

Comment:
```suggestion
* The browser receives an authorization code response from the Okta Authorization Server. If there is an error or the authentication fails for any reason, the browser returns the error to the end user. If the authentication is successful, the Okta Authorization Server issues an authorization code value and includes it in a redirect that sends the browser over to the client at the **Login Redirect URI** callback location.
```

Reply:
Leaving "the Client" as that this section discusses it as a Proper Noun. Other changes made.